src/components/datatable/DataTable.md

```diff
@@ -15,10 +19,49 @@ function Component() {
   const delimiters = { row: "\n", cell: "\t" };
 
   const options = {
+    pagination: pagination,
     page: 0,
     rowsPerPage: 10,
     rowsPerPageOptions: [10, 25, 50, 100],
+    onRowClick: (rowData) => {
+      const getRowData = rowData[1].props.tableMeta.rowData
+      const [chapter] = getRowData[2].split(delimiters.cell);
+      const [verse] = getRowData[3].split(delimiters.cell);
+      const [uid] = getRowData[4].split(delimiters.cell);
+      const rowId = `header-${chapter}-${verse}-${uid}`;
+      setSaveRowId(rowId)
+    },
+
+    onSearchClose: () =>{
+      const testRefId = saveRowId;
+      const element = document.getElementById(testRefId);
+      if ( element ) {
+        element.scrollIntoView();
+      } else {
+        alert(`Element id not found: ${testRefId}`);
+      }
+    },
+
+    onFilterChipClose: () =>{
+      console.log('dsddddd')
+      const testRefId = saveRowId;
+      const element = document.getElementById(testRefId);
+      if ( element ) {
+        element.scrollIntoView();
+      } else {
+        alert(`Element id not found: ${testRefId}`);
```

Comment:
I think console.log would be more appropriate in this case. Alerts are rather obtrusive was of displaying an error and in this case they really don't need to know an error occurred. 
```suggestion
        console.log(`Element id not found: ${testRefId}`);
```